### ISSUE TITLE
View for normal users

### ISSUE THREAD

**Issue by matss** [crispab/signup#25]
When adding support for pages that are protected by login of an admin user, large parts of the application will be inaccessible to the normal user. 
There should be a "view" where parts of the application that are only for admins should not be visible to normal users.
E.g. buttons like "edit event" should only be visible to admins.

---

One solution would be to go through the whole application and make sure components like edit buttons are removed from pages when the user is not logged in as an admin.
As the majority of the current SignUp application pages are intended for an admin user,
a much simpler solution would be to create a new view of an event where only participations can be edited.
Basically this corresponds to the pages:
/events/:id
/participations/:id/edit
/participations/new?eventId=eventId&userId=userId
Other pages are not needed by normal users
